infrastructure/api-mgmt.tf

```diff
@@ -46,3 +46,27 @@ module "api_mgmt_policy" {
   api_name               = "${module.api_mgmt.name}"
   api_policy_xml_content = "${replace(file("api-policy.xml"), "ALLOWED_CERTIFICATE_THUMBPRINTS", local.quoted_thumbprints)}"
 }
+
+module "api_mgmt_reconciliation" {
+  source        = "git@github.com:hmcts/cnp-module-api-mgmt-api?ref=master"
+  name          = "${var.component}-reconciliation-api"
+  api_mgmt_name = "core-api-mgmt-${var.env}"
+  api_mgmt_rg   = "core-infra-${var.env}"
+  revision      = "1"
+  product_id    = "${module.api_mgmt_product.product_id}"
+  display_name  = "Blob Router Reconciliation API"
+  path          = "reconciliation"
```

Comment:
would be nice to re-use same path - `reform-scan`. i believe name of module is what important - not the path. then we can have actual endpoint in the shape we actually want: `/reform-scan/reconciliation-report/{date}`

suggesting that because service endpoint is now better: `/reconciliation-report/{date}` ;)

Reply:
p.s. perhaps later on both APIs can be merged and marked as `revision = 2`. policy changes are in question here - not sure how to adapt to different endpoints but there must be some sort of filter